app/templates/jinja2/dashboard/volunteer/signup_players.html

```diff
@@ -49,6 +49,53 @@
       {% endfor %}
       </tbody>
     </table>
+    <table class="table table-striped">
+      <thead>
+      <tr>
+        <th scope="col">Self registration</th>
+        <th scope="col">Total registration</th>
+      </tr>
+      </thead>
+      <tbody>
+      <tr>
+        <td scope="col">{{ player_count - sum([loc.signupinvite_set.filter(used_at__isnull=False).count() for loc in locations]) }}</td>
+        <td scope="col">{{ player_count }}</td>
+      </tr>
+      </tbody>
+    </table>
+  {% endcall %}
+  {% call card("Signup Graph") %}
```

Comment:
This is part of the grid, so it'll be quite small. I'd move this outside of the grid in order to make it full-sized.